sdk/spring/azure-spring-cloud-autoconfigure/CHANGELOG.md

```diff
@@ -11,6 +11,10 @@
 ### Breaking Changes
 - Encode message payload with UTF-8 charset instead of default charset of JVM when the payload is String. ([#23056](https://github.com/Azure/azure-sdk-for-java/pull/23056))
 
```

Comment:
Ops, there is a piece of changelog for servicebus ignored here, could you help to add below content ?
### New Features
- Support configuration of `AmqpTransportType` for ServiceBusClientBuilder with property of `spring.cloud.azure.servicebus.transportType`, supported values are `AMQP` and `AMQP_WEB_SOCKETS`.